dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/DataManager.kt

```diff
@@ -27,7 +27,7 @@ class DataManager(
 ) : DataManagerInterface {
     var dataMetaInformationPerDataId = ConcurrentHashMap<String, DataMetaInformation>()
     val allDataTypes = DataTypesExtractor().getAllDataTypes()
-    private val greenAssetRatios = ConcurrentHashMap<StockIndex, BigDecimal>()
```

Comment:
Die Anpassungen am Green Asset Ratio hätten wir nicht gebraucht, oder? Von daher ist das auch in Ordnung, dass hier nur non financials berücksichtig wurden, richtig?

Reply:
ja das green asset ratio muss eigentlich raus. Es ist toter code.

Reply:
Dafür gibt es DALA-432. Können das aber sonnst auch jetzt schon löschen. Was meint Ihr?

Reply:
da es ein eigenständige Story ist und hier nicht stört würde ich es erstmal drin lassen und mich auf das abschließen dieser Story konzentrieren.

Reply:
Update: Durch die Änderung der Suchfunktion und dem DataType Enum im backend, hätte ich das GreenAssetRatio anpassen müssen. Habe es daher entfernt.

Reply:
Ok, dann ist das raus und hier mache ich zu.